addons/modules/config.cpp

```diff
@@ -0,0 +1,15 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        units[] = {"CBA_ModuleAttack","CBA_ModuleDefend","CBA_ModulePatrol"};
+        requiredVersion = 0.1;
+        requiredAddons[] = {"A3_Modules_F"};
+        //author[] = {"WiredTiger"};
```

Comment:
I believe it was the cause of an error when I was testing this morning. I'm guessing it has something to do with the update?